DigitalLearningSolutions.Web/Controllers/HomeController.cs

```diff
@@ -60,7 +65,7 @@ private LandingPageViewModel GetLandingPageViewModel(int sectionIndex)
             {
                 MiniHubNavigationModel = new MiniHubNavigationModel(LandingPageMiniHubName, sections, sectionIndex),
                 UserIsLoggedIn = User.Identity.IsAuthenticated,
-                CurrentSiteBaseUrl = configuration[ConfigHelper.CurrentSystemBaseUrlName],
+                CurrentSiteBaseUrl = configuration[ConfigHelper.CurrentSystemBaseUrlName]

```

Comment:
Is this post 624 merge? If so can you use my GetCurrentSystemBaseUrl method instead.

Reply:
I didn't actually touch this line, I think the formatter just removed a comma. Would you still like me to change it in this MR?

Reply:
Probably not worth it then.

Reply:
Added two editorConfig rules to accept trailing commas :)